.config/nvim/lua/plugins/packer.lua

```diff
@@ -0,0 +1,63 @@
+local present, packer = pcall(require, "packer")
+
+if not present then
+	return
+end
+
+return packer.startup(function()
+	use("wbthomason/packer.nvim") -- package manager
+
+	use("norcalli/nvim-colorizer.lua") -- css colors
+
+	-- File explorer
+	use({
+		"kyazdani42/nvim-tree.lua",
+		requires = { "kyazdani42/nvim-web-devicons" },
+	})
+
+	-- Themes
+	use("EdenEast/nightfox.nvim")
+	use({
+		"nvim-lualine/lualine.nvim",
+		requires = { "kyazdani42/nvim-web-devicons", opt = true },
+	})
+
+	-- LSP
+	use("neovim/nvim-lspconfig") -- Collection of configuration for LSP
+	use("williamboman/mason.nvim") -- Lagunage Server Auto installation
```

Comment:
typo

Reply:
that's the cool way to spell it, will update